internal/xdscache/v3/listener.go

```diff
@@ -383,6 +387,7 @@ func (c *ListenerCache) OnChange(root *dag.DAG) {
 				ServerHeaderTransformation(cfg.ServerHeaderTransformation).
 				NumTrustedHops(cfg.XffNumTrustedHops).
 				MaxRequestsPerConnection(cfg.MaxRequestsPerConnection).
+				PerConnectionBufferLimitBytes(cfg.PerConnectionBufferLimitBytes).
```

Comment:
There are two more places in this file where an `HTTPConnectionManagerBuilder` is used and needs this to be called (one for TLS, one for TLS+fallback)

Reply:
Updated